site/source/user-manual/backups/backup-setup/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _backup-setup:
+
+=============
+Setup Backups
+=============
+
+.. tip:: Keeping a strong password and making regular backups are the humble responsibilities of digital independence, remain ever vigilant!
```

Comment:
"Keeping a strong password" does not belong here.

Also, this wording is a bit hokie. How about a warning box saying "Create frequent backups to avoid loss of data!"